mk/spksrc.icon.mk

```diff
@@ -14,7 +14,7 @@ ICON_COOKIE = $(WORK_DIR)/.$(COOKIE_PREFIX)icon_done
 # Icons are only needed with "DSM UI configuration" (see spksrc.service.mk)
 ifneq ($(strip $(DSM_UI_DIR)),)
 ifneq ($(strip $(SPK_ICON)),)
-ifneq ($(strip $(SERVICE_PORT)),)
+ifneq ($(or $(strip $(SERVICE_PORT)), $(strip $(ADMIN_URL))),)
```

Comment:
Curious, what's this for?

Reply:
This is so that the icons are built for this package since there is no specific `SERVICE_PORT`. Didn't know if there was a better way to do this. Thoughts?

Reply:
This was subsequently enhanced in https://github.com/SynoCommunity/spksrc/pull/5649#issuecomment-1751843885.